pkgs/development/ocaml-modules/hacl-star/raw.nix

```diff
@@ -14,6 +14,8 @@ stdenv.mkDerivation rec {
 
   minimalOCamlVersion = "4.08";
 
+  hardeningDisable = [ "strictoverflow" ];
```

Comment:
It might be nice to add a comment to explain why this is here (e.g., what you wrote in the PR description).